mmv1/templates/terraform/custom_delete/firebase_app_check_service_config.go

```diff
@@ -0,0 +1,40 @@
+var obj map[string]interface{}
+log.Printf("[DEBUG] Deleting ServiceConfig %q", d.Id())
+
+project, err := tpgresource.GetProject(d, config)
+if err != nil {
+	return fmt.Errorf("Error fetching project for ServiceConfig: %s", err)
+}
+billingProject := project
+
+url, err := tpgresource.ReplaceVars(d, config, "{{FirebaseAppCheckBasePath}}projects/{{project}}/services/{{service_id}}")
+if err != nil {
+	return err
+}
+
+// Custom logic: add all mutable fields to the updateMask
+url, err = transport_tpg.AddQueryParams(url, map[string]string{"updateMask": "enforcementMode"})
+if err != nil {
+	return err
+}
+
+// err == nil indicates that the billing_project value was found
+if bp, err := tpgresource.GetBillingProject(d, config); err == nil {
+	billingProject = bp
+}
+
+res, err := transport_tpg.SendRequest(transport_tpg.SendRequestOptions{
+	Config:    config,
+	Method:    "PATCH",
+	Project:   billingProject,
+	RawURL:    url,
+	UserAgent: userAgent,
+	Body:      obj,
```

Comment:
I don't know how to read this. What is the request exactly?

Reply:
Yeah this is quite a mouthful. It roughly translates to 

`PATCH firebaseappcheck.googleapis.com/v1/projects/{{project}}/services/{{service_id}}?updateMask=enforcementMode
{} (empty body)

1. Method is PATCH
2. The `url` includes `updateMask` from line 16
3. The body `obj` is empty because it's never touched since declaration.